### PR TITLE
remove line return in ignition scripts

### PR DESCRIPTION
**Pull request by shawndwells**
Resolves https://github.com/ComplianceAsCode/content/issues/5653

Many of the ignition remediations created an erroneous new line entry.

Did /not/ re-encode anything, just removed the line returns.

### CODE REVIEW

**Comment by shawndwells** [comment]
@jhrozek this should remove the extra ``%0A``

**Comment by shawndwells** [comment]
@matusmarhefka getting a weird error from the "content-test-filtering":
`````
First time build. Skipping changelog.
[content-test-filtering-pr] $ /bin/bash /tmp/jenkins7610072178458973655.sh
python3: can't open file '/home/jenkins/workspace/content-test-filtering-pr/ctf/content_test_filtering.py': [Errno 2] No such file or directory
Build step 'Execute shell' marked build as failure
Setting status of ea70f864f6459d97ed9c2a9f7c92b933957f150b to FAILURE with url https://jenkins.complianceascode.io/job/content-test-filtering-pr/314/ and message: 'Build finished. '
Using context: content-test-filtering
Finished: FAILURE
`````

Any idea what this test is? Assuming it can be ignored.

**Comment by shawndwells** [comment]
@openscap-ci retest this please

File: linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/ignition/shared.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
     storage:
       files:
       - contents:
-          source: data:,%23%23%20First%20rule%20-%20delete%20all%0A-D%0A%0A%23%23%20Increase%20the%20buffers%20to%20survive%20stress%20events.%0A%23%23%20Make%20this%20bigger%20for%20busy%20systems%0A-b%208192%0A%0A%23%23%20This%20determine%20how%20long%20to%20wait%20in%20burst%20of%20events%0A--backlog_wait_time%2060000%0A%0A%23%23%20Set%20failure%20mode%20to%20syslog%0A-f%201%0A
```
Comment (jhrozek) [inline linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/ignition/shared.yml:11]: I'm afraid a mass-removal won't work. Each rule really has to be tested separately. E.g. here the audit_basic_configuration's check does expect a single newline. Before, there were two and my recent commit removed one of them.

A bigger issue is that some content is just incorrect, e.g. see PR#5651 where the urldecoded content would just be different from what the checks expect.

**Comment by matusmarhefka** [comment]
> @matusmarhefka getting a weird error from the "content-test-filtering":
> 
> ```
> First time build. Skipping changelog.
> [content-test-filtering-pr] $ /bin/bash /tmp/jenkins7610072178458973655.sh
> python3: can't open file '/home/jenkins/workspace/content-test-filtering-pr/ctf/content_test_filtering.py': [Errno 2] No such file or directory
> Build step 'Execute shell' marked build as failure
> Setting status of ea70f864f6459d97ed9c2a9f7c92b933957f150b to FAILURE with url https://jenkins.complianceascode.io/job/content-test-filtering-pr/314/ and message: 'Build finished. '
> Using context: content-test-filtering
> Finished: FAILURE
> ```
> 
> Any idea what this test is? Assuming it can be ignored.

I think @mildas already resolved it.

**Comment by shawndwells** [comment]
Closing since @jhrozek is manually reviewing each remediation.

**Pull request closed without merging**